src/components/Forms/CalculatorСalorieForm/validationSchema.js

```diff
@@ -2,22 +2,26 @@ import * as yup from 'yup';
 
 export const validationSchema = yup.object({
   height: yup
-    .number('Height it is number')
+    .number()
+    .typeError('You must specify a number')
```

Comment:
'Please specify a number'
@Yuliia-Lysiuk, I suggest to use easy form of wording 👍 :)